package/MDAnalysis/topology/core.py

```diff
@@ -41,35 +41,7 @@
     get_atom_mass, guess_atom_mass, guess_atom_charge,
     guess_bonds, guess_angles, guess_dihedrals, guess_improper_dihedrals,
 )
-from ..lib.util import cached
+from ..lib.util import cached, get_parser_for
```

Comment:
I've accidentally killed this submodule, but keeping the stub means nothing breaks.  Is it worth deprecating etc so we can eventually remove it?